scripts/task_test_aot_build_import.sh

```diff
@@ -28,6 +28,9 @@ else
   MAX_JOBS=$BASE_MAX_JOBS
 fi
 
+# Export MAX_JOBS for PyTorch's cpp_extension to use
```

Comment:
Nit: currently we are not relying on torch's cpp_extension, the environment variable is used for ninja.